src/eterna/index.ts

```diff
@@ -8,9 +9,11 @@ log.setLevel(isProduction ? 'info' : 'trace');
 declare global {
     interface Window {
         EternaApp: typeof EternaApp;
+        FoldingAPIApp: typeof FoldingAPIApp;
         app: EternaApp; // this syntax is used in index.html.tmpl, at least...
         __PIXI_APP__?: PIXI.Application;
     }
 }
 
 window.EternaApp = EternaApp;
+window.FoldingAPIApp = FoldingAPIApp;
```

Comment:
Ideally I'd use a different `index.ts` for `folding-api.html`, so that we don't have to import `pixi` in the script editor - I'm not sure what is the relationship between `eterna/index.ts` and `index.html` though.